packages/discovery-provider/plugins/pedalboard/apps/relay/src/middleware/logging.ts

```diff
@@ -30,7 +31,7 @@ export const outgoingLog = (request: Request, response: Response) => {
   const statusCode = response.statusCode;
   const path: string = route.path
   if (!path.includes("health")) {
-    logger.info(
+    response.locals.ctx.logger.info(
       { requestId, path, method, responseTime, statusCode },
```

Comment:
still needed? should the child logger already have some of these?

Reply:
good catch, thanks!